hledger/Hledger/Cli/BalanceView.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE OverloadedStrings, RecordWildCards #-}
+
+module Hledger.Cli.BalanceView (
+  BalanceView(..)
+ ,balanceviewmode
+ ,balanceviewReport
+) where
+
+import Control.Monad (unless)
+import Data.Time.Calendar (Day)
+import Data.List (intercalate)
+import Data.Monoid (Sum(..), (<>))
+import System.Console.CmdArgs.Explicit
+
+import Hledger
+import Hledger.Cli.Balance
+import Hledger.Cli.CliOptions
+
+data BalanceView = BV { bvmode    :: String
```

Comment:
Let's add explanatory haddocks for this type and functions.

I wouldn't mind using the full BalanceView name as constructor, in the interest of keeping the code clear and obvious for contributors. It's not used that often.